packages/venia-ui/package.json

```diff
@@ -41,6 +41,7 @@
     "lodash.throttle": "~4.1.1",
     "memoize-one": "~5.0.0",
     "prop-types": "~15.7.2",
+    "react-animate-height": "^2.0.23",
```

Comment:
Why was this dependency added?

Reply:
@michaelyu0123, just left that comment above the end of last week with James and it seems that some conflicts just came up that need to be resolved.

Reply:
this was added to create smooth dropdown for discount summary